openbmc-sfw

```diff
@@ -107,6 +178,24 @@ image_activate = subparsers.add_parser('activate', help='Activate input image id
 image_activate.add_argument('image_id', help='The image id to activate')
 image_activate.set_defaults(func=do_activate_image)
 
+image_update_auto = subparsers.add_parser('update_auto', help='Upload and '
+    + 'activate an image, and then reboot the BMC to apply the update '
+    + 'if necessary')
+image_update_auto.add_argument('image', help='The image to update to')
+image_update_auto.add_argument(
+    '--reboot',
+    dest='reboot',
+    action='store_true',
+    default=False,
+    help='Set if the BMC should reboot after the update')
+image_update_auto.add_argument(
+    '--no-reboot',
+    dest='reboot',
+    action='store_false',
+    default=False,
```

Comment:
Seems like we should make this the default

Reply:
You want to not reboot by default? Or you want to reboot by default? Right now it's set to not reboot by default.

Reply:
You want to reboot by default or NOT reboot by default? Right now default is set to not reboot.